service/models.py

```diff
@@ -31,6 +31,9 @@ class Condition(Enum):
     OPEN_BOX = 2
     USED = 3
 
+    # This is the last value in the enum and is meant to be a default value. Nothing should come after this
```

Comment:
Not sure if you're going to get any conflicts because I added this change already?